src/components/Header/index.jsx

```diff
@@ -27,7 +27,7 @@ const headerLogo64dpi = 'images/gc_' + angleMode + '_yellow_logo_64.png';
 const headerLogo56dpi = 'images/gc_' + angleMode + '_yellow_logo_56.png';
 const headerLogo56dpiNoText = 'images/gc_' + angleMode + '_yellow_logo_56_vert.png';
 
-const ForwardNavLink = forwardRef((props, ref) => <NavLink innerRef={ref} {...props} />);
+const ForwardNavLink = forwardRef((props, ref) => <NavLink {...props} />);
```

Comment:
I think this became unnecessary when we upgraded react-router past version 5.1, but no one has removed the unnecessary ref forwarding.

At this point, `ForwardNavLink` is at best useless and at worst silently swallowing the `ref`, so we should get rid of it and just pass a plain old `NavLink` where it was used.